opentracing/ext/tags.py

```diff
@@ -55,9 +60,12 @@
 # PEER_SERVICE (string) records the service name of the peer
 PEER_SERVICE = 'peer.service'
 
-# PEER_HOSTNAME records the host name of the peer
+# PEER_HOSTNAME (string) records the host name of the peer
 PEER_HOSTNAME = 'peer.hostname'
 
+# PEER_ADDRESS (string) records the ip address of the peer
```

Comment:
the `address` is more general than the ip address... please look at https://github.com/opentracing/specification/blob/master/semantic_conventions.md for details.